src/css/utils/_position.scss

```diff
@@ -10,3 +10,11 @@
   @extend .top-0;
   @extend .left-0;
 }
+
+.top-neg-10 {
```

Comment:
For negative positioning the Tailwind naming convention would be -top-10 and -left-51, check here https://tailwindcss.com/docs/top-right-bottom-left

Reply:
Fixed in above-mentioned PR.